### PR TITLE
 HTML skeleton created

### PR DESCRIPTION
**Pull request by Tyna1992**
HTML skeleton created

### CODE REVIEW

**Review by ep-cc — CHANGES_REQUESTED** [review]
Thanks!

See my comments inline.

File: HTML/NumberGuessingGame.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Number guessing game</title>
+    <script src=""></script>
```
[the remark this answers is not in the document]
Comment (Tyna1992) [reply HTML/NumberGuessingGame.html:6]: The script file implementation belongs to another colleague.

File: HTML/NumberGuessingGame.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+<h1>Number guessing game</h1>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (Tyna1992) [reply HTML/NumberGuessingGame.html]: Done!

**Review by ep-cc — APPROVED** [review]
LGTM!

**Pull request merged**